tests/MongoDBDriverManager_executeQuery_error-001.phpt

```diff
@@ -0,0 +1,60 @@
+--TEST--
+PHPC-924: Cursor::setTypeMap() may unnecessarily convert first BSON document (type map)
```

Comment:
LGTM with this removed, then. As you noted, the regression tests for the projection crash fixed by the libmongoc bump is in PHPC.